index.html

```diff
@@ -22,7 +21,7 @@
     <!-- build:js assets/bundle.js -->
     <!-- bower:js -->
     <script src="/bower_components/jquery/dist/jquery.js"></script>
-    <script src="/bower_components/bootswatch-dist/js/bootstrap.js"></script>
+    <script src="/bower_components/bootstrap-sass/assets/javascripts/bootstrap.js"></script>
```

Comment:
don't need to modify this file, the link and script tags will be generated by `gulp wiredep`,
if it's not, then the package config is still wrong